src/Liquidator_NEW.sol

```diff
@@ -178,8 +178,9 @@ contract Liquidator_NEW is Owned {
     ///////////////////////////////////////////////////////////////*/
 
     function liquidateAccount(address account) external nonReentrant {
-        // Store the initiator address
-        address initiator = msg.sender;
+        // Store the initiator address and set the inAuction flag to true.
+        auctionInformation[account].initiator = msg.sender;
+        auctionInformation[account].inAuction = true;
```

Comment:
It is now possible to overwrite ongoing Liquidations.
Don't we need a line like:
`if (auctionInformation[account].inAuction) revert Liquidator_AuctionOngoing();`
As was in the old Liquidator?